recipes/librdkafka/all/conanfile.py

```diff
@@ -111,6 +112,12 @@ def generate(self):
 
     def build(self):
         apply_conandata_patches(self)
+        # There are references to libcrypto.lib and libssl.lib in rdkafka_ssl.c for versions >= 1.8.0
+        if Version(self.version) >= "1.8.0" and is_msvc(self) and \
+                self.settings.build_type == "Debug" and self.options.get_safe("ssl", False):
+            rdkafka_ssl_path = os.path.join(self.source_folder, "src", "rdkafka_ssl.c")
+            replace_in_file(self, rdkafka_ssl_path, "libcrypto.lib", "libcryptod.lib")
+            replace_in_file(self, rdkafka_ssl_path, "libssl.lib", "libssld.lib")
```

Comment:
It's fragile. What will happen if we decide to rename these lib names (this d postfix shouldn't have been added to openssl libs in openssl recipe, there were many discussions about this issue).

You should refer to cpp_info.libs values of openssl, so that it can be future proof.